kolibri/plugins/device/assets/src/views/UserPermissionsPage/index.vue

```diff
@@ -1,107 +1,112 @@
 <template>
 
-  <div>
-    <h1 v-if="user === null">
-      {{ $tr('userDoesNotExist') }}
-    </h1>
+  <ImmersivePage
+    v-if="!$store.state.core.loading"
```

Comment:
This conditional is the only thing that I can see that is different between this page and the other pages here - can't think what else might be causing the issues in the permissions selection reported by @pcenov.